examples/shader/custom_vertex_attribute.rs

```diff
@@ -87,8 +87,13 @@ impl RenderAsset for CustomMaterial {
         (render_device, material_pipeline): &mut SystemParamItem<Self::Param>,
     ) -> Result<Self::PreparedAsset, PrepareAssetError<Self::ExtractedAsset>> {
         let color = Vec4::from_slice(&extracted_asset.color.as_linear_rgba_f32());
+
+        let byte_buffer = [0u8; Vec4::SIZE.get() as usize];
```

Comment:
Given that Size is now a part of our "global public api", I'd prefer it if this was `ShaderSize::SHADER_SIZE`. `SIZE` feels too generic / too easy to conflate with "size of Vec4 type on the CPU".

Reply:
I agree with the `ShaderSize` rename however I'm not sure about `ShaderSize::SHADER_SIZE` vs `ShaderSize::SIZE` yet. What do you think? If `SHADER_SIZE` is not a hard requirement, maybe we could go with the re-export `Size as ShaderSize` for now?

Reply:
Definitely not a hard requirement! The type alias is perfectly fine for this pr. The rename from Size->ShaderSize does make it a bit more self documenting on import, but in the actual code I still think `X::SIZE` is a liability. SIZE is just too generic of a concept. Imo the extra clarity of `X::SHADER_SIZE` would benefit all encase users (not just bevy).

Reply:
It's a solid argument. I used to use the full disambiguation syntax (i.e. `<x as encase::Size>::SIZE`) but later removed since it's not really ergonomic. I'll rename encase's `Size::SIZE` to `ShaderSize::SHADER_SIZE` in the next major release; let me know if you'd like the change to happen sooner since I don't have an ETA on that.

Reply:
Awesome. Much appreciated! Definitely no rush. I'd prefer it if it happened before Bevy 0.8 (mid july) to avoid breaking people twice, but even if that doesn't work out its not the biggest deal in the world.